### ISSUE TITLE
Can't specify params when running inference on MLServer

### ISSUE THREAD

**Issue by svpino** [SeldonIO/MLServer#1927]
When deploying an MLflow model using MLServer, we can't specify `params` as part of the request.

Here is the error returned by the server:

```
{"error":"mlflow.utils.proto_json_utils.MlflowInvalidInputException: Invalid input. One of \"instances\" and \"inputs\" must be specified (not both or any other keys).Received: ['inputs', 'params']"}%
```

Here is how I'm running the server:

```
mlflow models serve -m runs:/<run_id>/model -p 5000 --enable-mlserver
```

Here is an example of the request:

```
curl -X POST http://0.0.0.0:8080/invocations \
    -H "Content-Type: application/json" \
    -d '{"inputs": [{
            "sex": "MALE"
        }], 
        "params": {"capture": true}}'
```

It seems that MLServer doesn't support parameters as part of the request, which breaks its compatibility with MLflow models that require these parameters to work.

**Comment by sakoush** [comment]
@svpino this has been fixed for the standard mlflow runtime `/infer` endpoint as part of: https://github.com/SeldonIO/MLServer/pull/1921

Would it be possible to use this endpoint instead of `/invocations`, otherwise we would welcome contributions similar to the above.

Note that  `/invocations` endpoint is a custom endpoint in mlserver and lacks support for a lot of features from mlserver compared to `/infer`.

**Comment by svpino** [comment]
Unfortunately, MLflow requires an `/invocations` endpoint, so having it on `/infer` only will not solve the compatibility issue.

**Comment by sakoush** [comment]
@svpino is this something that you would like to fix in a PR? We encourage contributions from the community,

**Comment by svpino** [comment]
I'd love to. I'll try to get to this at some point, but I can't make any promises for now.

**Comment by dubovikmaster** [comment]
Hello @svpino! Any updates here?

**Comment by svpino** [comment]
> Hello [@svpino](https://github.com/svpino)! Any updates here?

Not really, sorry. Didn't have time to fix this and I moved on to something else.

**Comment by dubovikmaster** [comment]
Is there any hope that you will get around to this in the near future?

**Comment by svpino** [comment]
Probably, I won't. At least, I don't have plans to do this for now.

**Comment by dubovikmaster** [comment]
@svpino 
But this seems like a major oversight.
The mlflow documentation recommends using MLServer for industrial deployment of ML models.
And here it turns out that you can't use parameters in queries, which is critical for deploying LLM models, for example.(

**Comment by svpino** [comment]
@dubovikmaster Yes, I agree this is a very important feature. Please, understand I was the person who reported it. I'm not a contributor to this project. I think you have the impression that I am somehow responsible for fixing it. I'm not.
